fastjet/internal/delaunay/point.go

```diff
@@ -210,3 +214,26 @@ func (l location) String() string {
 		panic(fmt.Errorf("delaunay: unknown location %d", int(l)))
 	}
 }
+
+type points []*Point
+
+// remove removes given points from a slice of points.
+//
+// remove will remove all occurrences of the points.
+func (ps points) remove(pts ...*Point) points {
+	points := make(points, 0, len(ps))
```

Comment:
too many `points` :)
I would call the returned value `out`,  `o` or `opoints`.

Reply:
done